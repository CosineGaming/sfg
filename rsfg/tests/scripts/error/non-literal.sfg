fn main()
	var x = 5
	panic(x, x)

