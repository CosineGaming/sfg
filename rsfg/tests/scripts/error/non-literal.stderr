[ERROR] literal value required for panic at 3:8
[ERROR] literal value required for panic at 3:11